src/pages/leejiyon/Main/Components/Article.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from 'react';
+import Comments from './Comments';
+import './Article.scss';
+
+class Article extends Component {
+  inputRef = React.createRef();
+  state = {
+    isArticleLiked: false,
+    isCommentInputEntered: false,
+    commentInputValue: '',
+    currentUser: 'hao',
+    lastPostedCommentTIme: '42분 전',
+    commentLikeNum: '9',
+    userComments: [
+      {
+        id: 1,
+        userId: 'iu',
+        comment: '아자아자',
+        showTrashBtn: false,
+        liked: true,
+      },
+      {
+        id: 2,
+        userId: 'jiyon',
+        comment: '공부를 하자',
+        showTrashBtn: false,
+        liked: false,
+      },
+      {
+        id: 3,
+        userId: 'Shaman_king',
+        comment: '벨로그 쓰기',
+        showTrashBtn: false,
+        liked: false,
+      },
+    ],
+    id: 4,
+  };
+
+  onClickArticleHeartBtn = () => {
+    this.setState({
+      isArticleLiked: !this.state.isArticleLiked,
+    });
+    this.setState({
+      commentLikeNum: this.state.isArticleLiked ? 9 : 10,
+    });
+  };
+
+  onClickPostCommentInput = e => {
+    if (e.keyCode === 13) {
+      e.preventDefault();
+      this.onClickPostBtn();
+    }
+  };
+
+  onChangeCommentInput = e => {
+    this.setState({
+      commentInputValue: e.target.value,
+    });
+    if (e.target.value.length > 0) {
+      this.setState({
+        isCommentInputEntered: true,
+      });
+    } else {
+      this.setState({
+        isCommentInputEntered: false,
+      });
+    }
+  };
+
+  onClickPostBtn = () => {
+    if (!this.state.commentInputValue.length) {
+      return;
+    }
+    this.setState({
+      userComments: [
+        ...this.state.userComments,
+        {
+          id: this.state.id,
+          userId: this.state.currentUser,
+          comment: this.state.commentInputValue,
+          showTrashBtn: true,
+        },
+      ],
+      commentInputValue: '',
+      id: this.state.id + 1,
+      lastPostedCommentTIme: '방금',
+    });
+    this.inputRef.current.focus();
+  };
+
+  onClickCommentHeartBtn = userComment => {
+    let currentClickedCommentIdx = this.state.userComments.indexOf(userComment);
+    let userComments = [...this.state.userComments];
+    let user = {
+      ...userComment,
+    };
+    if (user.liked) {
+      user.liked = false;
+    } else {
+      user.liked = true;
+    }
+    userComments[currentClickedCommentIdx] = user;
+    this.setState({ userComments });
+  };
+
+  onClickDeleteBtn = v => {
+    this.setState({
+      userComments: this.state.userComments.filter(user => {
+        return user.id !== v.id;
+      }),
```

Comment:
```suggestion
      userComments: this.state.userComments.filter(user => user.id !== v.id)
```
arrow function에서 return이 바로 이루어질 경우에는 중괄호와 return keyword 생략 가능합니다!
별개로 v 라는 매개변수가 어떤 의미인지 명확히 안들어오네요 적절한 네이밍 사용해주세요!

Reply:
네 꼼꼼하게 봤어야 했는데  습관적으로 v로 한 것 같습니다.
clickedComment로 수정했습니다 !